### PR TITLE
Remove jQuery from PreviewThemeChat, Blog, MessageComposer #1812

### LINKED ISSUE #1812: Remove jQuery from PreviewThemeChat, Blog.react.js, and MessageComposer
**Actual Behaviour**

Currently jQuery is present in PreviewThemeChat, Blog.react.js, and MessageComposer

**Expected Behaviour**

Remove jQuery

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1812 
Changes: 

Closed https://github.com/fossasia/chat.susi.ai/pull/1854 due to conflicts

1. Created **getBlogReponse**, **getSusiPreviewReply** using axios. **getSusiPreviewReply** is created for preview theme chat component.
2. Removed jQuery from MessageComposer, shifting urlParam to helper function.

Fixes this also:
screenshot 2018-12-21 at 6 07 10 pm

Demo Link: https://pr-1870-1-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
Travis error: Aborted - you do not have permission to publish to pr-1870-fossasia-susi-web-chat.surge.sh

File: src/components/Blog/Blog.react.js [hunk 1]
```diff
@@ -90,25 +89,16 @@ class Blog extends Component {
...
-        //eslint-disable-next-line
-        api_key: BLOG_KEY, // put your api key here
-        count: 50,
-      },
-    }).done(
-      function(response) {
-        if (response.status !== 'ok') {
-          throw response.message;
+    getBlogReponse()
+      .then(({ payload }) => {
```
Comment (akshatnitd) [inline src/components/Blog/Blog.react.js]: I am not sure if this destructuring is required.

**Comment by AakashMallik** [comment]
@akshatnitd  SIr the blog is not working because the server is not able to respond with the keys on time and the current implementation is synchronous which ideally should have been async. This blog issue has been resolved in my PR https://github.com/fossasia/chat.susi.ai/pull/1851. I would be fixing merge conflict with this PR anyway, so it won't be an issue.

**Comment by AakashMallik** [comment]
```
import $ from 'jquery';
import urls from './utils/urls';

const url = `${urls.API_URL}/aaa/getApiKeys.json`;

/* global module */

$.ajax({
  url: url,
  dataType: 'json',
  crossDomain: true,
  timeout: 3000,
  async: true,
}).done(function(output) {
  module.exports.MAP_KEY = output.keys.MAP_KEY;
  module.exports.CAPTCHA_KEY = output.keys.CAPTCHA_KEY;
  module.exports.BLOG_KEY = output.keys.BLOG_KEY;
});

```

This is the current config file implementation, and as you can guess it will return undefined if the keys are not fetched by the time the blog route is called.

**Comment by fragm3** [comment]
@AakashMallik is right, the API key is not being fetched.

**Comment by AakashMallik** [comment]
@akshatnitd Sir the rss2json api responds even if the key is undefined but if you send a **count** param, then it would respond saying that this parameter needs the blogkey for this param to work. I was fiddling with their api and got to know about this. But not sure why 200. :man_shrugging: Anyway, fixing the merge conflict should fix the blog issue here.

**Comment by fragm3** [comment]
These merge conflicts are killing me

**Pull request closed without merging**